packages/core/src/writers/single-mode.ts

```diff
@@ -132,3 +138,15 @@ export const writeSingleMode = async ({
     throw `Oups... 🍻. An Error occurred while writing file => ${e}`;
   }
 };
+
+const generateImports = (
```

Comment:
This function should be common because it is used in all modes.
You can create a file like `generate-imports-for-builder` in the same directory.
There is already a function with the same name in `utils`, so I want to name the function `generateImportsForBuilder` to avoid confusion.